CHANGELOG.md

```diff
@@ -9,6 +9,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 - Fix notifications muted state for the current user in channel members [#3236](https://github.com/GetStream/stream-chat-swift/pull/3236)
 - Reset channel members and watchers state when fetching the initial state of the channel [#3245](https://github.com/GetStream/stream-chat-swift/pull/3245)
 - Fix inconsistent message text when extremely quickly updating it [#3242](https://github.com/GetStream/stream-chat-swift/pull/3242)
+### 🔄 Changed
+- Set background mapping enabled (prepare data on a separate queue when database changes) [#3250](https://github.com/GetStream/stream-chat-swift/pull/3250)
```

Comment:
We should try to avoid adding implementation details in the Changelog, it is better just to describe the end goal:
```suggestion
- Enable background mapping by default, which improves performance overall [#3250](https://github.com/GetStream/stream-chat-swift/pull/3250)
```